### PR TITLE
RishabhBajaj97 ISSUE # 3764

### PR DESCRIPTION
**Pull request by RishabhBajaj97**

### CODE REVIEW

File: brands/shop/beauty.json [hunk 1]
```diff
@@ -174,5 +174,15 @@
       "name": "Yves Rocher",
       "shop": "beauty"
     }
+  },
+  "shop/beauty| Carlance": {
```
Comment (bmillemathias) [inline brands/shop/beauty.json:178]: no space after |

File: brands/shop/beauty.json [hunk 2]
```diff
@@ -174,5 +174,15 @@
...
+    "countryCodes": ["fr"],
+    "tags": {
+      "beauty" : "hair;nails;skin_care;massage",
+      "brand" : "Carlance",
+      "brand:wikidata" : "Q91601625",
+      "name" : "Carlence",
```
Comment (bmillemathias) [inline brands/shop/beauty.json:184]: This is Carlance, not Carlence

File: brands/shop/beauty.json (hunk 1 above)
Comment (RishabhBajaj97) [reply brands/shop/beauty.json:178]: hello, thanks for the feedback!

I have added my email and name, thank you.

so for this change do i need to again branch and create a new PR?
I think I forgot to run  "npm run build"

File: brands/shop/beauty.json (hunk 2 above)
Comment (RishabhBajaj97) [reply brands/shop/beauty.json]: I think I have successfully done the changes..please let me know

Thanks

**Pull request merged**